base/libgit2/libgit2.jl

```diff
@@ -469,7 +479,27 @@ function revcount(repo::GitRepo, fst::AbstractString, snd::AbstractString)
     return (fc-1, sc-1)
 end
 
-""" git merge [--ff-only] [<committish> | FETCH_HEAD] """
+"""
+    merge!(repo::GitRepo; committish::AbstractString="", branch::AbstractString="", fastforward::Bool=false, merge_opts::MergeOptions=MergeOptions(), checkout_opts::CheckoutOptions=CheckoutOptions()) -> Bool
+
+Perform a git merge on the repository `repo`, merging commits
+with diverging history into the current branch. Returns `true`
+if the merge succeeded, `false` if not.
+
+The optional arguments are:
+  * Merge the named commit(s) in `committish`.
+  * Merge the branch `branch` and all its commits since it diverged from the current branch.
+  * If `fastforward` is `true`, only merge if the merge is a fast-forward (the current branch
+    head is an ancestor of the commits to be merged), otherwise refuse to merge and return
+    `false`. This is equivalent to the Git CLI option `--ff-only`.
```

Comment:
Inconsistent capitalization of "Git"; it appears in lowercase everywhere else (though technically it probably should be capitalized everywhere since it's a proper noun)